src/entry.sh

```diff
@@ -14,7 +14,7 @@ RESTART="${RESTART:-1}"
 TELNETPORT="${TELNETPORT:-7072}"
 CONFIGTYPE="${CONFIGTYPE:-"fhem.cfg"}"
 DNS=$( cat /etc/resolv.conf | grep -m1 nameserver | sed -e 's/^nameserver[ \t]*//' )
-export DOCKER_GW="${DOCKER_GW:-$(ip -4 route list match 0/0 | cut -d' ' -f3)}"
```

Comment:
Why would you mix this change into the Travis>GHActions migration path?

Reply:
Also, why is this method more reliable and what made you changing this command? pls explain so we have it documented as well.

Reply:
I've changed this, to avoid a additional package dependency in the image. The `ip` command comes via `iproute2`
It's mixed this in here, because this package isn't in the install list, so i changed to available commands.